clients/mongoDbClient.js

```diff
@@ -0,0 +1,23 @@
+const { MongoClient } = require('mongodb');
+const uri = "mongodb://lrqi_db:lrqi_db_pwd@lrqidb-shard-00-00-wksjy.mongodb.net:27017,lrqidb-shard-00-01-wksjy.mongodb.net:27017,lrqidb-shard-00-02-wksjy.mongodb.net:27017/test?ssl=true&replicaSet=LRQIDB-shard-0&authSource=admin&retryWrites=true";
```

Comment:
use process.env.CONNECTION_STRING here, and we can put the connection string in environment variables or manifest.yml file